communitiesuk/funding-service-pre-award
# Assess frontend

Adds the frontend on assess to:
- Accept questions (just the button) - this is to be used for the 1/0 scoring
- Request changes - GET and POST endpoints

Things left to be done:
- Set sub-criteria status to Changes Requested if they exist
- Display changes requested flag differently

### How to test
- pull branch locally
- `make pre up`
- Create an application on Apply f.x. CTDF application Pro tip: use lead_assessor example email
- Go to assess -> application -> subcriteria
- You will see the Accept/Request change button

### Screenshots
Accept / Request changes buttons:
![image](https://github.com/user-attachments/assets/83605520-f63d-4e91-aecf-7d3ced721f65)

Request changes form:
![image](https://github.com/user-attachments/assets/41d09429-5161-4862-90e5-0279e4f8866a)

Display change requests on assess:
![image](https://github.com/user-attachments/assets/11e4d611-66fa-455f-b402-b27527b97071)

Display CHANGE_REQUEST status on tasklist:
![image](https://github.com/user-attachments/assets/92eb6e8b-d296-4df8-8583-6fbc75d9364b)

Display CHANGE_RECEIVED status on tasklist:
![image](https://github.com/user-attachments/assets/639cd9d3-2b09-499c-9a17-0611b3e3c8a2)

Display CHANGE_REQUEST and CHANGE_RECEIVED statuses on applications page:
![image](https://github.com/user-attachments/assets/db6a40be-d502-4895-aba9-eb377adb10d5)
